sktime/forecasting/trend/_polynomial_trend_forecaster.py

```diff
@@ -178,6 +249,13 @@ def get_test_params(cls, parameter_set="default"):
                 "regressor": RandomForestRegressor(),
                 "degree": 2,
                 "with_intercept": False,
+                "prediction_intervals": False,
+            },
+            {
+                "regressor": RandomForestRegressor(),
+                "degree": 2,
+                "with_intercept": True,
+                "prediction_intervals": True,
```

Comment:
We are not testing prediction intervals in absence of intercept, may be add one more test case?

Reply:
good catch, let's add that too, as there is a special branch in the `_predict_interval` that is otherwise not executed in the standardized tests